src/components/SecondaryPanes/Frames/tests/FrameMenu.js

```diff
@@ -0,0 +1,73 @@
+const originalL10N = L10N;
+import FrameMenu from "../FrameMenu";
+import { showMenu } from "devtools-launchpad";
+import { copyToTheClipboard } from "../../../../utils/clipboard";
+jest.mock("devtools-launchpad", () => ({ showMenu: jest.fn() }));
+jest.mock("../../../../utils/clipboard", () => ({
+  copyToTheClipboard: jest.fn()
+}));
+
+function generateMockCopySource(label, accesskey) {
```

Comment:
this could be a nice general purpose helper for mocking a menu item.

With that said, I'm concerned we might be testing too much here. For instance, in this case I'd be happy to just check the IDs are what we expect:

`expect(items.map(i => i.d)).toEqual(["joe", "bob", "dan"])`, this tells us that the menu has the correct items and we can later test other properties that might be important like conditional disabled states...